public/app/features/alerting/unified/components/receivers/form/fields/TemplateSelector.tsx

```diff
@@ -202,26 +195,27 @@ function TemplateSelector({ onSelect, onClose, option, valueInForm }: TemplateSe
           <Stack direction="column" gap={1}>
             <Stack direction="row" gap={1} alignItems="center">
               <Select<Template>
+                data-testid="existing-templates-selector"
                 placeholder="Choose template"
                 aria-label="Choose template"
                 onChange={(value: SelectableValue<Template>, _) => {
-                  setTemplate(value?.value);
+                  setTemplate(value);
                 }}
                 options={options}
                 width={50}
-                value={template ? { label: template.name, value: template } : undefined}
+                defaultValue={defaultTemplateValue}
               />
               <IconButton
                 tooltip="Copy selected template to clipboard. You can use it in the custom tab."
-                onClick={() => copyToClipboard(getUseTemplateText(template?.name ?? ''))}
+                onClick={() => copyToClipboard(defaultTemplateValue?.value?.content ?? template?.value?.content ?? '')}
```

Comment:
I feel it depends. I'm a bit confused about the naming of these tabs `Select existing template` and `Enter custom message`. They seem exclusive (because of the radio selector), but copying the name suggests they are somehow related

I'm not sure what's the intention of this copy button. Do we expect the user to copy the title or content and paste it into the custom message? Why would they want to do that?

Reply:
` Select existing template` and `Enter custom message` are exclusive, you can either use one or the other for the value of the field you're editing. But you can use the first one, to check which template you want to use in the custom tab....
The intention of this copy button is to be able to copy the 'usage of the template' , nor the content